explainpowershell.analysisservice/MetaData.cs

```diff
@@ -22,26 +23,25 @@ public static class MetaData
         [FunctionName("MetaData")]
         public static IActionResult Run(
             [HttpTrigger(AuthorizationLevel.Anonymous, "get", Route = null)] HttpRequest req,
-            [Table(HelpTableName)] CloudTable cloudTable)
+            [Table(HelpTableName)] TableClient client)
```

Comment:
Now that you've shown me `CloudTable` was replaced by this `TableClient` concept, how did you find that?

Reply:
I implemented ITableEntity for the model classes, but it was incompatible with TableQuery, as the namespace and such changed. Then I moved those services to also use Azure.Data.Tables, but then CloudTable and TableQuery were missing. At that point I googled "is tablequery deprecated" -> https://www.google.com/search?q=is+tablequery+deprecated

The top hit (https://stackoverflow.com/questions/68511912/what-package-should-i-use-for-azure-tables-when-migrating-away-from-windowsazur) describes exactly what I ran in to